src/components/contact/Contact.tsx

```diff
@@ -0,0 +1,106 @@
+import { PawHubContainer } from "components/layout/Grid/PetCardFlex";
+import { SubmitHandler, useForm } from "react-hook-form";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+import "./Contact.css";
+import { EmailInput, MessageInput, NameInput } from "./Inputs";
+
+export interface FormData {
+  names: string;
+  email: string;
+  message: string;
+}
+
+const Contact = () => {
+  const {
+    register,
+    handleSubmit,
+    reset,
+    formState: { errors },
+  } = useForm<FormData>();
+
+  const onSubmit: SubmitHandler<FormData> = (data) => {
+    // TODO: tie the success/error notification to the respective submission states
+    if (data) {
+      reset();
+      successNotification();
+      return;
+    }
+
+    errorNotification();
+
+    console.log(data);

```

Comment:
can you remove the console.log

Reply:
Definitely, let me work on it. Thanks for the feedback.